README.md

```diff
@@ -292,10 +292,12 @@ Joplin implements the SQLite Full Text Search (FTS4) extension. It means the con
 Search type | Description | Example
 ------------|-------------|---------
 Single word | Returns all the notes that contain this term. | For example, searching for `cat` will return all the notes that contain this exact word. Note: it will not return the notes that contain the substring - thus, for "cat", notes that contain "cataclysmic" or "prevaricate" will **not** be returned.
-Multiples words | Returns all the notes that contain **all** these words, but not necessarily next to each other. | `dog cat` - will return any notes that contain the words "dog" and "cat" anywhere in the note, no necessarily in that order nor next to each other. It will **not** return results that contain "dog" or "cat" only.
-Phrase query | Add double quotes to return the notes that contain exactly this phrase. | `"shopping list"` - will return the notes that contain these **exact terms** next to each other and in this order. It will **not** return for example a note that contains "going shopping with my list".
+Multiple word | Returns all the notes that contain **all** these words, but not necessarily next to each other. | `dog cat` - will return any notes that contain the words "dog" and "cat" anywhere in the note, no necessarily in that order nor next to each other. It will **not** return results that contain "dog" or "cat" only.
+Phrase | Add double quotes to return the notes that contain exactly this phrase. | `"shopping list"` - will return the notes that contain these **exact terms** next to each other and in this order. It will **not** return for example a note that contains "going shopping with my list".
 Prefix | Add a wildcard to return all the notes that contain a term with a specified prefix. | `swim*` - will return all the notes that contain eg. "swim", but also "swimming", "swimsuit", etc. IMPORTANT: The wildcard **can only be at the end** - it will be ignored at the beginning of a word (eg. `*swim`) and will be treated as a literal asterisk in the middle of a word (eg. `ast*rix`)
-Field restricted | Add either `title:` or `body:` before a note to restrict your search to just the title, or just the body. | `title:shopping`, `body:egg`
+Title | Prefix your query with `title:` to search note titles for matches. | `title:shopping`
+Body | Prefix your query with `body:` to search note content for matches. | `body:egg`
+*combination* | Combine the above types as needed. | `title:shop*`, `body:"shopping list"`
```

Comment:
Does combination actually work? Otherwise I like your changes but if you could confirm this that would be great.

Reply:
Ok, I don't think it does. I'll revert that last change and resubmit.

Reply:
not sure how a README change can result in build failure. Rebased onto head, still same. I assume the problem is upstream.